python/tvm/api.py

```diff
@@ -73,22 +74,24 @@ def max_value(dtype):
     return _api_internal._max_value(dtype)
 
 
-def const(value, dtype):
+def const(value, dtype=None):
```

Comment:
Hey, why make it optional? As a compiler, we should be more serious about which type we would feed into the IR framework.